region/maridia/inner-pink/Crab Shaft.json

```diff
@@ -446,7 +446,13 @@
         {"notable": "Ice Only Crab Climb"},
         "h_canNavigateUnderwater",
         "canTrickyUseFrozenEnemies",
-        "canCarefulJump"
+        "canCarefulJump",
+        {"or": [
+          "canTwoTileSqueeze",
+          "canTrickyJump",
```

Comment:
I wonder if `canSunkenTileWideWallClimb` should be an option. Seems a fair bit harder to get it positioned to where you can jump up without a WJ. Not sure what the `canTwoTileSqueeze` option is for.

Reply:
The twoTileSqueeze was to climb high enough to jump around the pipe, but that's still hard to do.  I was using two crab to climb the room to avoid any awkward jumps, but in races we do canSunkenTileWideWallClimb all the time here.